xml/bp_troubleshooting_objectgateway.xml

```diff
@@ -13,143 +13,141 @@
-     <filename>.asok</filename> file from another user. If an <filename>.asok</filename>
-     file from another user exists and there is no running pid, remove the
-     <filename>.asok</filename> file and try to start the process again.
-     This may occur when you start the process as a root user and the
-     startup script is trying to start the process as a <literal>www-data</literal> or
-     <literal>apache</literal> user and an existing <filename>.asok</filename> is
-     preventing the script from starting the daemon.
-   </para>
-   <para>
-     The &ogw; init script (<filename>/etc/init.d/radosgw</filename>) also has a verbose argument
-     that can provide some insight as to what could be the issue:
+     The most basic health check to test a running &ogw; process is to simply
+     point your browser or client at the &ogw; endpoint. This should return
+     an empty bucket results for the anonymous user:
    </para>
 <screen>
-/etc/init.d/radosgw start -v
+&prompt.smaster;curl -I
 </screen>
    <para>
-     Or:
+     If the result returns a 405 status, this indicates MethodNotAllowed.
+     The output will result in a NoSuchBucket XML file. If the result returns
+     a 404 status, this means the &ogw; DNS name is misconfigured or the
+     requests were not made at an endpoint resolving to &ogw; DNS name.
+   </para>
+ </sect1>
+ <sect1 xml:id="troubleshooting-gateway-notrunning">
+   <title>The Gateway is Not Running</title>
+   <para>
+     Usually a restarted gateway (usually automatic under systemd) should
+     restore the service. This can be achieved via:
    </para>
 <screen>
-/etc/init.d radosgw start --verbose
+&prompt.cephuser;ceph orch daemon restart <replaceable>rgw-daemon-name</replaceable>
 </screen>
- </sect1>
- <sect1 xml:id="bp-troubleshooting-http-request">
-   <title>HTTP Request Errors</title>
    <para>
-     Examining the access and error logs for the web server itself is
-     probably the first step in identifying what is going on. If there is
-     a 500 error, that usually indicates a problem communicating with
-     the radosgw daemon. Ensure the daemon is running, its socket path
-     is configured, and that the web server is looking for it in the
-     proper location.
+     For bumping up the log level, under the <literal>client.<replaceable>rgw-name</replaceable></literal>
+     section, the &ogw; configurable can be increased from 1 (default)
+     up to 20 (very verbose). For the messages sent to the cluster itself the
+     messenger debug levels can be raised (off by default). This is controlled
+     via <literal>debug ms</literal>. Usually a level of 1 is sufficient to
+     gather enough information. For injecting arguments in a running process:
    </para>
+<screen>
+&prompt.cephuser;ceph-daemon <replaceable>client.rgw</replaceable> config set debug_rgw 20
+</screen>
    <para>
-     Check a cluster's status:
+     Or, alternatively, using <command>ceph-daemon</command>:
```

Comment:
move the using `ceph-daemon` to the previous line (before the screen), Let this one be just Or, alternatively, using the `ceph` cli